client/my-sites/domains/domain-management/transfer/transfer-page/index.tsx

```diff
@@ -356,7 +356,7 @@ const TransferPage = ( props: TransferPageProps ) => {
 
 		return (
 			<Card className="transfer-page__advanced-transfer-options">
-				<CardHeading size={ 16 }>{ __( 'Advanced Options' ) }</CardHeading>
+				<CardHeading size={ 16 }>{ __( 'Transfer to another registrar' ) }</CardHeading>
```

Comment:
This new copy makes much more sense.

Should we add the old strings as compatibility or are you planning to leave it open with string-freeze?

![image](https://github.com/Automattic/wp-calypso/assets/1044309/d2ffc7a8-b981-4a1c-9cfa-9d0adb653284)

Reply:
Good point, I'll put a fallback in